WLogger/Repository/Commons/ImageFetcher.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ImageFetcher.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/28/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+public struct  ImageFetcher {
+    
+    public func fetchImage(imageURL: NSURL) -> SignalProducer<UIImage?, NSError> {
```

Comment:
Then you can define a `typealias ImageFetcher = NSURL -> SignalProducer<UIImage, NSError>`